api.swagger.yaml

```diff
@@ -2966,7 +2980,22 @@ components:
       type: object
       properties:
         access_threshold_users:
-          description: 'users who accessed the site on 4 or more days within a month'
+          description: 'Users who accessed the site on 4 or more days within a month'
+          type: integer
+          example: 500
+    user-metrics-profiles-schema:
+      type: object
+      properties:
+        user_count:
+          description: 'The total number of active users'
```

Comment:
Can we add a definition of "active" users, or a reference to where the definition can be found? (bearing in mind it may be collapsed in the swagger view)